src/WorkspaceFileResolver.cpp

```diff
@@ -49,8 +49,13 @@ const TextDocument* WorkspaceFileResolver::getTextDocumentFromModuleName(const L
     if (Luau::startsWith(name, "untitled:"))
         return getTextDocument(Uri::parse(name));
 
-    if (auto filePath = platform->resolveToRealPath(name))
-        return getTextDocument(Uri::file(*filePath));
+    if (auto filePath = platform->resolveToRealPath(name)) {
+		auto it = managedFiles.find(filePath->generic_string());
+        if (it != managedFiles.end()) {
+            return &it->second;
+		}
+		return getTextDocument(Uri::file(*filePath));
+	}
```

Comment:
indentation is messed up! use spaces not tabs

Reply:
Sorry about that, in a bit of a rush